style.css

```diff
@@ -40,4 +41,25 @@ td{
     -webkit-transition: 0.2s;
     transition: opacity 0.2s;
     border-radius: 12px;
+}
+
+.buttonContainer{
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    align-content: space-between;
+}
+
+.btn{
```

Comment:
Hi Junior, In line #54 in the CSS file, I think that would be better to use button element for the styles intead to use class btn because using button element we work with the lowest specificity and follow the recomendation that Carlos said in class.

Reply:
Thanks Rodrigo, I will change it shortly.